applications/API/forms.py

```
@@ -0,0 +1,291 @@
+                raise ValidationError(
+                    _("Last name is required when not submitting anonymously"),
+                    code="last-name-required",
+                )
+
+        # Validate contact preference
+        if post_review_contact_desired and not reporter_email:
+            raise ValidationError(
+                _("Email address is required"),
+                code="email-required-for-contact",
+            )
+            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)